packages/eslint-config-airbnb/rules/style.js

```diff
@@ -34,6 +34,8 @@ module.exports = {
     'lines-around-comment': 0,
     // disallow mixed 'LF' and 'CRLF' as linebreaks
     'linebreak-style': 0,
+    // specify the maximum length of a line in your program
```

Comment:
can you add https://github.com/eslint/eslint/blob/master/docs/rules/max-len.md in a comment?